sdk/core/core-http-compat/src/extendedClient.ts

```diff
@@ -50,4 +76,49 @@ export class ExtendedServiceClient extends ServiceClient {
       });
     }
   }
+
+  /**
+   * Compatible send operation request function.
+   *
+   * @param operationArguments
+   * @param operationSpec
+   * @returns
+   */
+  async sendOperationRequest<T>(
+    operationArguments: OperationArguments,
+    operationSpec: OperationSpec
+  ): Promise<T & ModifiedResponseObject> {
+    this.userProvidedCallBack = operationArguments?.options?.onResponse;
+
+    if (!operationArguments.options) operationArguments.options = {};
+    operationArguments.options.onResponse = this.onResponse.bind(this);
+
+    const result: T = await super.sendOperationRequest(operationArguments, operationSpec);
+
+    // Object.defineProperty(result, "_response", {
```

Comment:
These commented lines also work fine and produce the same result as the lines 103 - 108. @xirzec Do you have any preference?

Reply:
Done